spec/support/clean_solr.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 RSpec.configure do |config|
   config.before(:each, clean_index: true) do
-    client = Hyrax.index_adapter.connection
+    client = Valkyrie::IndexingAdapter.find(:solr_index).connection
```

Comment:
again I feel like we should leave it open indexer wise.

Reply:
you can definitely make the case for a tag ability to use the null indexer

Reply:
how about https://github.com/samvera/hyrax/pull/4175/files#r362945930
??